crates/orchestrator/src/config.rs

```diff
@@ -218,7 +218,7 @@ impl<K: SignatureKey, E: ElectionConfig> NetworkConfig<K, E> {
         file: Option<String>,
     ) -> (NetworkConfig<K, E>, NetworkConfigSource) {
         if let Some(file) = file {
-            error!("Retrieving config from the file");
+            println!("Retrieving config from the file");
```

Comment:
Do we want to make these raw print statements? Would it be better to just info log or something?

Reply:
Yes! Because I'd like to have them as `info` messages instead of `error`, however datadog does not show `info` or `debug` level message :( and datadog categorize `println` as `info`, therefore I changed to this.